src/completed.js

```diff
@@ -0,0 +1,33 @@
+function updateTaskStatus(status, taskIndex, tasksLocal) {
+  for (let i = 0; i < tasksLocal.length; i += 1) {
+    if (tasksLocal[i].index === (taskIndex)) {
+      tasksLocal[i].completed = status;
+    }
+  }
+  localStorage.setItem('tasks', JSON.stringify(tasksLocal));
+  return tasksLocal;
+}
+
+function arrangeIndexes(tasksLocal) {
+  for (let i = 0; i < tasksLocal.length; i += 1) {
+    tasksLocal[i].index = (i + 1);
+  }
+  return tasksLocal;
+}
+
+function deleteCompletedTasks(tasksLocal) {
+  const taskCheckbox = document.querySelectorAll('.checkbox');
+  taskCheckbox.forEach((cb) => {
+    if (cb.checked) {
+      const li = cb.parentNode;
+      li.remove();
+    }
+  });
+  tasksLocal = tasksLocal.filter((tr) => tr.completed !== true);
+  // tasksLocal = tasksLocal.filter(!tasksLocal.completed);
```

Comment:
- [ ] Please kindly delete this commented line of code to have a clean codebase, it is very important to have clean code 😉 😉